edxlearndot/learndot.py

```diff
@@ -142,11 +176,10 @@ def sort_enrolments_by_expiry(enrolment_list):
         ValueError: if a sorting date can't be parsed
         OverflowError: if a sorting date can't be fit into the largest valid C integer
     """
-
-    return sorted(enrolment_list, key=extract_enrolment_sort_key, cmp=compare_enrolment_sort_keys)
+    return sorted(enrolment_list, key=functools.cmp_to_key(extract_and_compare_enrolment_sort_keys))
 
 
-class EnrolmentStatus(object):
+class EnrolmentStatus(object):  # pylint: disable=useless-object-inheritance
```

Comment:
nit: python3 classes all inherit from object, so no need to disable the pylint:

```suggestion
class EnrolmentStatus:
```

Reply:
Yeah, I completely forgot about that when I removed the Python 2.7 stuff I did. I kept the `(object)` for python 2.7, but now it isn't important.